src/k8s/api/v1/types.go

```diff
@@ -170,6 +180,35 @@ func (c ClusterStatus) String() string {
 		result.WriteString("  spare-nodes: none\n")
 	}
 
+	return result.String()
+}
+
+// TODO: Print k8s version. However, multiple nodes can run different version, so we would need to query all nodes.
+func (c ClusterStatus) String() string {
+	result := strings.Builder{}
+
+	// Status
+	if c.Ready {
+		result.WriteString("status: ready")
+	} else {
+		result.WriteString("status: not ready")
+	}
+	result.WriteString("\n")
+
+	// High availability
+	result.WriteString("high-availability: ")
+	if c.HaClusterFormed() {
```

Comment:
nit: can you make `HaClusterFormed` also private - we don't need it elsewhere and I probably made it public by mistake.